daemon/graphdriver/overlay2/overlay.go

```diff
@@ -577,7 +590,7 @@ func (d *Driver) Get(id, mountLabel string) (_ containerfs.ContainerFS, retErr e
 	for i, s := range splitLowers {
 		absLowers[i] = path.Join(d.home, s)
 	}
-	opts := fmt.Sprintf("lowerdir=%s,upperdir=%s,workdir=%s", strings.Join(absLowers, ":"), path.Join(dir, "diff"), path.Join(dir, "work"))
+	opts := indexOff + "lowerdir=" + strings.Join(absLowers, ":") + ",upperdir=" + path.Join(dir, "diff") + ",workdir=%s" + path.Join(dir, "work")
```

Comment:
No need to convert this from fmt.Sprint

Reply:
To me, it looks more readable that way (as the option arguments are right there next to options), and `Sprintf()` does nothing but string concatenating here.

Using `fmt.Sprintf()` in Golang to merely concatenate a few strings (without doing anything else like padding, quoting etc.) looks like a coding pattern that came straight from C, where you can't concatenate more than 2 strings at once, and should be discouraged unless necessary.

Concatenation is also quite faster (not that it matters in this particular case).

Reply:
@dmcgowan please let me know what you think about ^^^ , and I'll surrender and obey :)

Reply:
This change is fine, neither has great readability but not worth making it an array and joining